mapusaurus/geo/views.py

```diff
@@ -23,12 +24,30 @@ def tract_centroids_in_json_format(request):
     return response % ', '.join(geo.tract_centroids_as_geojson() for geo in censusgeos)
 
 def get_censustract_geoids(request):
-    geos = get_censustract_geos(request)
+    if 'neLat' in request.GET:
+        geos = get_censustract_geos(request)
+    else:
+        geos = get_tracts_by_msa(request)
     return geos.values_list('geoid', flat=True)
 
-def get_censustract_geos(request):
+def get_tracts_by_msa(request):
```

Comment:
Is it worth it to create one get_tracts function that handles both bounds and msa? If bounds(neLat, neLon swLat, swLon) are passed in, then use bounds else use 'metro'.